local-cli/bundle.js

```diff
@@ -0,0 +1,59 @@
+var http = require('http');
+var fs = require('fs');
+var path = require('path');
+var blacklist = require('../packager/blacklist.js');
+var ReactPackager = require('../packager/react-packager');
+
+var OUT_PATH = 'iOS/main.jsbundle';
+
+function getBundle(flags) {
+
+  var options = {
+    projectRoots: [path.resolve(__dirname, '../../..')],
+    transformModulePath: require.resolve('../packager/transformer.js'),
+    assetRoots: [path.resolve(__dirname, '../../..')],
+    cacheVersion: '2',
+    blacklistRE: blacklist('ios')
```

Comment:
we can revisit this later when Android ships, no need for extra CLI option for now (you had this on your TODO comment)